### PR TITLE
Add SwaggerUI for `todo-csharp-cosmos-sql`

### LINKED ISSUE #50: [Issue] deployed service api link not working for `todo-csharp-cosmos-sql`
- [x] Make sure you've installed the latest version using [instructions in the wiki](../wiki/install)

**Output from `todo-csharp-cosmos-sql`**
Run `todo-csharp-cosmos-sql` and copy and paste the output here:
image

**Describe the bug**
deployed service api link (e.g https://app-api-4ncptg7bwnti2.azurewebsites.net/) is 404 (page not found).

**To Reproduce**
Run `azd up --template todo-csharp-cosmos-sql`

**Expected behavior**
Add a handler for the root path to the C# version of the API

**Environment**
Information on your environment:
Windows, AzCli 2.38.0

### PR DESCRIPTION
**Pull request by hemarina**
Fix #50 
Serve `SwaggerUI` from the root of the API endpoint

### CODE REVIEW

File: templates/todo/api/js/tsconfig.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "es5",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "target": "es2015",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```
Comment (ellismg) [inline templates/todo/api/js/tsconfig.json:14]: FWIW - I didn't end up needing this change locally and still had a good VS Code experience - I think.  I know you were running into some promise issues without this, but everything seemed to work for me end to end without it.  Could you maybe remove this and see if things still work?

File: templates/todo/api/csharp-cosmos-sql/Program.cs [hunk 2]
```diff
@@ -26,5 +26,16 @@
     policy.AllowAnyMethod();
 });
 
+// Swagger UI
+app.UseSwaggerUI(options => {
+    options.SwaggerEndpoint("./openapi.yaml", "v1");
```
Comment (ellismg) [inline templates/todo/api/csharp-cosmos-sql/Program.cs:31]: For this to work, we need to have `openapi.yaml` placed into the `wwwroot` folder, and I suspect we need to modify the `repo.yaml` files for this project (and the mongo version) to ensure this file is copied to the right place:

https://github.com/Azure/azure-dev/blob/c414fc573d4785e6f9d875e3529ceb5ad0a2f974/templates/todo/projects/csharp-mongo/repo.yaml#L36-L40

Something around here needs to be updated.

**Review by ellismg — CHANGES_REQUESTED** [review]
I think we need some small changes to some `repo.yaml`'s for this to work.

File: templates/todo/api/js/tsconfig.json (hunk 1 above)
Comment (karolz-ms) [reply templates/todo/api/js/tsconfig.json:14]: FWIIW, we use `es2020` for the VS Code extension. There is really no good reason to stick with old JS flavors if when the project is a node backend app.

File: templates/todo/api/js/tsconfig.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "es5",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "target": "es2022",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```
Comment (karolz-ms) [inline templates/todo/api/js/tsconfig.json:14]: This is too ambitious. Node 16 (which is current LTS) does not support all Ecmascript 2022 features: https://node.green/. 

Your safe+modern bet is `es2020`

**Review by ellismg — APPROVED** [review]
LGTM

**Pull request merged**